### PR TITLE
Add Prettier

### LINKED ISSUE #56: [FEATURE] Setup Prettier
### Description

Setup Prettier for formatting the code consistently

### Screenshots

_No response_

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by Cahllagerfeld**
closes #56

### CODE REVIEW

**Comment by Cahllagerfeld** [comment]
fire event

File: package.json [hunk 1]
```diff
@@ -42,10 +44,19 @@
   },
   "devDependencies": {
     "autoprefixer": "10.4.5",
+    "husky": "^8.0.2",
```
Comment (Cahllagerfeld) [inline package.json:47]: Also added husky to run prettier in the pre-commit hook

Comment (Cahllagerfeld) [reply package.json:48]: Prettier run for all files

**Comment by eddiejaoude** [comment]
Great idea 👍 thanks Cahl

**Pull request merged**